server/SwipeDecision.py

```diff
@@ -15,16 +16,66 @@ def getPotentialMatchList(currentUserId):
             cursor.execute(PotentialMatchQuery, userID)
             potentialListId = [i[0] for i in cursor.fetchall()]
 
+            newPotentialMatchList = []
+
+            if(len(potentialListId) != 0):
+
+                findLocation = "SELECT Users.longitude, Users.latitude, Profile.maxDistance FROM Users INNER JOIN Profile ON Users.id=Profile.userId WHERE Users.id = %s"
+                cursor.execute(findLocation, userID)
+                currentUserLocation = cursor.fetchone()
+                currentUserLongitude = readLocation(currentUserLocation)[0]
```

Comment:
I'm not sure why you would need a separate method to do this. It doesn't decrease the number of lines you're writing and it doesn't split up the code nicely.